suite-common/wallet-utils/src/exportTransactions.ts

```diff
@@ -115,46 +125,107 @@ const makePdf = (
         });
     });
 
-const prepareContent = (data: Data) => {
-    const { type, transactions, coin } = data;
-    return transactions.map(formatAmounts(coin)).map(t => {
-        let addresses = [];
-        if (t.tokens.length > 0) {
-            addresses = t.tokens.map(token => {
-                if (token?.address && token?.amount) {
-                    return `${token.address} (${token.amount} ${token.symbol.toUpperCase()})`;
-                }
+const prepareContent = (data: Data): Fields[] => {
+    const { transactions, coin } = data;
+    return transactions
+        .map(formatAmounts(coin))
+        .flatMap(t => {
+            const sharedData = {
+                date: new Intl.DateTimeFormat('default', dateFormat).format(
+                    (t.blockTime || 0) * 1000,
+                ),
+                time: new Intl.DateTimeFormat('default', timeFormat).format(
+                    (t.blockTime || 0) * 1000,
+                ),
+                timestamp: t.blockTime?.toString(),
+                type: t.type.toUpperCase(),
+                txid: t.txid,
+            };
 
-                return null;
-            });
-        } else {
-            addresses = t.targets.map(target => {
+            if (t.tokens.length > 0) {
+                return t.tokens.map((token, index) => {
+                    if (token?.address && token?.amount) {
+                        return {
```

Comment:
nit: would be clearer to return `null` if `!token?.address || !token?.amount` to have the positive result in the end